tsconfig.build.json

```diff
@@ -6,7 +6,7 @@
     "allowSyntheticDefaultImports": true,
     "moduleResolution": "node",
     "module": "esnext",
-    "lib": ["esnext"],
+    "lib": ["esnext", "dom", "dom.iterable"],
```

Comment:
Is it necessary to specify `dom` and `dom.iterable`? Aren't these automatically included?

If we must have this, we'll also need to update `cwp-template-aws` package. See `packages/cwp-template-aws/template/common/tsconfig.build.json`

Also, existing projects will need a project upgrade (a codemod) to apply this change, once people upgrade to 5.38.
Let's write this down as a new task, because it will be in a different repository.

Reply:
Ok so we need a codemod then. Let's keep the `lib` option, and specify those in all projects.